packages/react/src/primitives/Breadcrumbs/Breadcrumbs.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from 'react';
+import classNames from 'classnames';
+
+import {
+  BaseBreadcrumbProps,
+  BreadcrumbsProps,
+  Primitive,
+  ForwardRefPrimitive,
+} from '../types';
+import { ComponentClassNames } from '../shared/constants';
+import { View } from '../View';
+import { BreadcrumbItem } from './BreadcrumbItem';
+import { BreadcrumbLink } from './BreadcrumbLink';
+import { BreadcrumbsContext } from './BreadcrumbsContext';
+
+const BreadcrumbsPrimitive: Primitive<BreadcrumbsProps, 'nav'> = (
+  { className, children, separator = '/', ...rest },
+  ref
+) => {
+  const componentClasses = classNames(
+    ComponentClassNames.Breadcrumbs,
+    className
+  );
+  const value = React.useMemo(() => {
+    return {
+      separator,
+    };
+  }, [separator]);
+
+  return (
+    <View
+      as="nav"
+      aria-label="Breadcrumb"
+      className={componentClasses}
+      ref={ref}
+      {...rest}
```

Comment:
Is it intentional that `rest.as` and `rest.aria-label` override the specified values above?